app/views/layouts/simulator.html.erb

```diff
@@ -4,6 +4,12 @@
     <title>CircuitVerse - Digital Circuit Simulator online</title>
     <%= csrf_meta_tags %>
     <script src="/js/pace.js"></script>
+    <%= javascript_include_tag "i18n" %>
+    <%= javascript_include_tag "translations", skip_pipeline: true %>
```

Comment:
`javascript_include_tag` is a rails thing, we don't want to use this. But not sure.
CC @tachyons

Reply:
I checked this what this tag is doing is creates html javascript tag ```<script src="/assets/i18n.23232323...."></script>```

same file is generated by middleware at ```/public/javascripts/i18n.js```  if i use ```<script src="/javascripts/i18n.js"></script>```
It's working perfectly fine

Reply:
![2021-05-27_19-02](https://user-images.githubusercontent.com/55848322/119835135-2f64ee80-bf1e-11eb-9765-29620c0b6064.png)